conda/cli/main_info.py

```diff
@@ -247,7 +248,11 @@ def execute(args, parser):
         envs=[],
         python_version='.'.join(map(str, sys.version_info)),
         requests_version=requests_version,
+        user_agent=user_agent,
     )
+    if not on_win:
+        info_dict['UID'] = os.getuid()
```

Comment:
I think this should be `os.geteuid()`

ref: http://stackoverflow.com/questions/14950378/what-is-difference-between-os-getuid-and-os-geteuid